crate/src/sdk/mod.rs

```diff
@@ -1165,6 +1177,95 @@ impl Sdk {
         to_js_result(reward)
     }
 
+    pub async fn query_notes_to_spend(
+        &self,
+        owner: String,
+        chain_id: String,
+    ) -> Result<JsValue, JsError> {
+        let xvk = ExtendedViewingKey::from_str(&owner)?;
+        let viewing_key = ExtendedFullViewingKey::from(xvk).fvk.vk;
+
+        let mut shielded: ShieldedContext<masp::JSShieldedUtils> = ShieldedContext::default();
+        shielded.utils.chain_id = chain_id.clone();
+        // TODO: pass handler
+        shielded.try_load(async |_| {}).await;
```

Comment:
Just curious: What would a handler do here? Is it just important that we await `try_load` for now?